opentelemetry-tracing/opentelemetry-java-tracing/spring-boot-kotlin/postgres/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
```

Comment:
Why do we need docker-compose for 1 service?

Reply:
It's taken directly from the SFx example.